lib/tests/formulae_fetch.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+module Homebrew
+  module Tests
+    class FormulaeFetch < TestFormulae
+      attr_accessor :testing_formulae
+
+      def run!(args:)
+        info_header "Testing formulae:"
+        puts testing_formulae
+        puts
+
+        testing_formulae.each do |formula_name|
+          fetch_formula!(formula_name, args: args)
+          puts
+        end
+      end
+
+      private
+
+      def fetch_formula!(formula_name, args:)
+        formula = Formula[formula_name]
+        tags = formula.bottle_specification.collector.tags
+
+        tags.each do |tag|
+          test "brew", "fetch", "--retry", "--formulae", "--bottle-tag=#{tag}", formula_name
+        end
```

Comment:
Does this allow fetching them all on Linux?

Reply:
Yes, this does.

Reply:
@carlocab very cool 🎉 again, probably worth noting in docs.

Reply:
Added this to the docs.